### PR TITLE
Add Slide Beacon

### PR DESCRIPTION
**Pull request by dermike**
Share links from your Mac using this app to broadcast them as a Physical Web/Eddystone URL bluetooth beacon or mDNS.
https://github.com/dermike/electron-slide-beacon

### CODE REVIEW

**Comment by sindresorhus** [comment]
The readme could use some improvements. After reading the readme, I have no idea what I would use it for or how to use it. I start the app, select a link, and start broadcasting, but how do I receive the URL? It's also unclear in the app, what "Use reveal.js presentation plugin" means or how I can use it. The app should also notify me if bluetooth is off and I turn to turn it on, ideally offer to turn it on for me. The app should follow the macOS human interface guidelines when it comes to the menubar menus. `Stop broadcasting` should be a disabled menubar item when not broadcasting.

File: readme.md [hunk 1]
```diff
@@ -148,6 +148,7 @@ Made with Electron.
 - [Wire](https://github.com/wireapp/wire-desktop) - Messenger and calling app.
 - [Ramme](https://github.com/terkelg/ramme) - Unofficial Instagram app.
 - [Moonview](https://github.com/teesloane/moonview) - Ambient notepad.
+- [Slide Beacon](https://github.com/dermike/electron-slide-beacon) - Physical Web beacon broadcast app for Macs.
```
Comment (sindresorhus) [inline readme.md]: And I would call the repo: `slide-beacon-app`, not `electron-slide-beacon`.

**Comment by dermike** [comment]
Great feedback, thanks!

**Comment by sindresorhus** [comment]
Any updates on this?

**Comment by dermike** [comment]
Readme improvement and HIG fix is in the works, but repo name will stay since it's linked in a few places out of my control.

**Comment by MarshallOfSound** [comment]
@dermike If you change the repo name github will automatically forward the old URL to the new one.

https://github.com/blog/1508-repository-redirects-are-here

**Comment by dermike** [comment]
Everything except the bluetooth off notification should be fixed now. The module used to broadcast waits until it's enabled instead of warning, and I'm hesitant to use another module just for detecting it in this case, even though it's certainly good for the UX.

**Comment by sindresorhus** [comment]
Can you fix the merge conflict here?

**Comment by dermike** [comment]
Hopefully good to go now

**Comment by sindresorhus** [comment]
Perfect

**Pull request merged**